packages/api/src/blob-parse/optimism.ts

```diff
@@ -28,5 +32,11 @@ export function parseDecodedData(data: string): OptimismDecodedData | null {
     return null;
   }
 
+  const hash = await getFullBlockHash(decoded.data.l1OriginBlockHash);
+
+  if (hash) {
+    decoded.data.l1OriginBlockHash = hash;
+  }
```

Comment:
It would be nice to log errors here if the full hash is not found.

Reply:
Done [396b9e9](https://github.com/Blobscan/blobscan/pull/608/commits/396b9e92cc44f824aa877d504469d54debb0b628)

Reply:
@luis-herasme  should be an error, not a warning

Reply:
I've fixed it now